pkg/processor/trigger/http/types.go

```diff
@@ -43,8 +43,19 @@ type Configuration struct {
 
 	// Used to disable port publishing for the HTTP trigger on docker platform
 	DisablePortPublishing bool
+
+	Mode TriggerMode `json:"mode,omitempty"`
```

Comment:
Not sure why we don't use json annotations for the rest of the attributes, but if it doesn't break the behavior we should align all to have it.

Reply:
@TomerShor added them + checked that everything works as it used to